src/SFA.DAS.Roatp.CourseManagement.Application/ProviderStandards/Commands/DeleteProviderCourse/DeleteProviderCourseCommand.cs

```diff
@@ -8,12 +7,13 @@ public class DeleteProviderCourseCommand : IRequest
         public int Ukprn { get; }
         public int LarsCode { get; }
         public string UserId { get; set; }
-
-        public DeleteProviderCourseCommand(int ukprn, int larsCode, string userId)
+        public string UserDisplayName { get; set; }
+        public DeleteProviderCourseCommand(int ukprn, int larsCode, string userId, string userDisplayName)  
         {
             Ukprn = ukprn;
             LarsCode = larsCode;
             UserId = userId;
+            UserDisplayName = userDisplayName;
```

Comment:
I am wondering why there was no changes in any of the unit tests, can you make sure that this has coverage.